lib.sh

```diff
@@ -705,14 +722,12 @@ or when a new version of the app is released with the following command:
         if [ -d "$NC_APPS_PATH/$1" ]
         then
             occ_command app:enable "$1"
-            chown -R www-data:www-data "$NC_APPS_PATH"
```

Comment:
Please don't remove this line, it's there for a reason.

Reply:
okay, in my testing this is not needed (when installing apps through the store), but if you say so...

Reply:
this is reverted